packages/core/base/src/models/paymentElement.ts

```diff
@@ -1,14 +1,10 @@
 import { CheckoutEventMap } from "./events";
 import { Currency, Locale, PaymentMethod, UIConfig } from "./types";
 
-export type Recipient =
-    | {
-          email: string;
-          mintTo?: string;
-      }
-    | {
-          mintTo: string;
-      };
+export type Recipient = {
+    email?: string;
```

Comment:
`email` or `emailTo`?

i had `emailTo` in crossmint-main, but can change

Reply:
yeah `emailTo` works